compiler/compiler.go

```diff
@@ -1435,7 +1435,8 @@ func (b *builder) createFunctionCall(instr *ssa.CallCommon) (llvm.Value, error)
 			return b.emitSV64Call(instr.Args)
 		case strings.HasPrefix(name, "(device/riscv.CSR)."):
 			return b.emitCSROperation(instr)
-		case strings.HasPrefix(name, "syscall.Syscall"):
+		case strings.HasPrefix(name, "syscall.Syscall") || strings.HasPrefix(name, "syscall.RawSyscall"):
+			// TODO: implement b.createRawSyscall for real to reduce overhead slightly
```

Comment:
Which overhead? TinyGo just calls syscalls directly without any stack switching overhead like in upstream Go (in fact, the call is inlined so there's no real way to optimize it further). So this comment seems misleading.

Also, note that `b.createRawSyscall` is a different 'raw' than `syscall.RawSyscall`.

Reply:
Removed comment.